analyze/analyses/paths/paths_emit.rs

```diff
@@ -280,3 +92,209 @@ impl traits::Emit for Paths {
         Ok(())
     }
 }
+
+/// This module contains helper functions and structs used by the `emit_text`
+/// method in Path's implementation of the `traits::Emit` trait.
+mod emit_text_helpers {
+    use std::iter;
+
+    use twiggy_ir::Items;
+    use twiggy_opt::Paths;
+
+    use analyses::paths::paths_entry::PathsEntry;
+
+    /// This structure represents a row in the emitted text table. Size, and size
+    /// percentage are only shown for the top-most rows.
+    pub(super) struct TableRow {
+        pub size: Option<u32>,
+        pub size_percent: Option<f64>,
+        pub name: String,
+    }
```

Comment:
I *believe* `pub(super)` is what I should be using here. If that looks wrong though, lmk!